src/desktop.md

```diff
@@ -277,6 +278,27 @@ build output on other machines for testing purposes.
 
 #### Windows
 
+For building Windows executables, you can either use tooling to construct an
+MSIX installer, or you can build your own installer that collects 
+the components together.
+
+##### MSIX Packaging
+
+[MSIX][] is Microsoft's Windows app package format that provides a modern 
+packaging experience to all Windows apps. This format can either be used 
+to ship applications to Microsoft's Windows Apps store, or distribute 
+application installers directly.
+
+The easiest way to create an MSIX distribution for a Flutter project is to use
+the [`msix` pub package][msix package]. For an example of using the `msix` package
+from a Flutter desktop app, see the [Desktop Photo Search sample][].
+
+[MSIX]: https://docs.microsoft.com/en-us/windows/msix/overview
+[msix package]: {{site.pub}}/packages/msix
+[Desktop Photo Search sample]: {{site.github}}/flutter/samples/tree/master/experimental/desktop_photo_search
+
+##### Building your own installer for Windows
```

Comment:
These instructions aren't for building an installer -- they're for creating a zip file (which is a good thing to do but the title should reflect that).

Reply:
Done